src/style.css

```diff
@@ -1,3 +1,78 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
```

Comment:
I it is not a good idea to use * selector for anything but box sizing property.
please move font-family and font-weight to HTML tag and reset margins and padding for those elements you need.
for instance

```css
ul,
h1,
p,
// ... other selectors
h2 {
margin: 0;
padding: 0;
}
```

Reply:
@anastasiia-tilikina but in readme.md file says about reseting browser default margins
How i can make it correctly?
<img width="1066" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/156140967/412a6355-50ce-431a-87a7-7cb5a3a631f0">